src/coreComponents/constitutive/fluid/multifluid/CO2Brine/CO2BrineFluid.cpp

```diff
@@ -235,14 +240,16 @@ void CO2BrineFluid< PHASE1, PHASE2, FLASH >::postInputInitialization()
 
   string const expectedGasPhaseNames[] = { "CO2", "co2", "gas", "Gas" };
   m_p2Index = PVTFunctionHelpers::findName( m_phaseNames, expectedGasPhaseNames, viewKeyStruct::phaseNamesString() );
-
   createPVTModels();
 }
 
+/**
+ * @brief Create a PVT Model and output them
+ * @param isClone If we are in the case of a clone of a constitutive mode, never the output
+ */
```

Comment:
Why is there documentation in here? It goes in the HPP.
Plus, there is no longer a isClone param.